data_collection/gazette/spiders/pr_prudentopolis.py

```diff
@@ -0,0 +1,49 @@
+import datetime
+
+from dateparser import parse
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class PrPrudentopolis(BaseGazetteSpider):
+    TERRITORY_ID = "4120606"
+    name = "pr_prudentopolis"
+    start_date = datetime.date(2010, 5, 27)
+    allowed_domains = ["prudentopolis.pr.gov.br", "l2fsistemasweb.com.br"]
+    start_urls = ["https://www.prudentopolis.pr.gov.br/diario-oficial/"]
+
+    def parse(self, response):
+        start_url = "https://www.prudentopolis.pr.gov.br/diario-oficial/"
+        download_url = "https://www.prudentopolis.pr.gov.br"
+
+        lines = response.xpath("//tbody/tr")
+
+        urls = []
+        for relative_url in lines:
+            url = relative_url.xpath("td[4]/a/@href").extract()[0]
```

Comment:
A lib parsel sugere o uso de `get()` and `getall()` ao invés do extract(). Nesse caso você poderia fazer:

```suggestion
            url = relative_url.xpath("td[4]/a/@href").get()
```